Dockerfile

```diff
@@ -14,6 +14,9 @@ RUN wget -O /tmp/zeppelin-${ZEPPELIN_VER}-bin-all.tgz http://archive.apache.org/
 
 EXPOSE 8080 8443
 
+# Install GDAL utilities
+RUN apt-get install -y gdal-bin
```

Comment:
Could you move this up in the file. My mental model for containers is that everything is installed before you get to exposing ports/volumes. Obviously under the zeppelin installation so you don't break the cache for the slow step.